plugins/collaajUpdatePlugin/lib/model/collaajInstallData.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+// TODO: remove when done
+//require_once ("/opt/kaltura/app/plugins/collaajUpdatePlugin/lib/model/iniReader.php");
+// This class handles all the data queries relevant for colaaj and the ini file
+class collaajInstallData
+{
+
+//returnFilteredByOs - done as returnFilteredByExtension
+//returnNewerVersion - not needed
+//getLatestVersionData - done
+//returnVersionUpdateInfo - done
+
+
+    // private members  ///////////////////////////////////////////////////////////////////////////////////////
+    private $ini_handle = "";  // handle to the ini object
+    private $ini_content = "";
+    private $ini_sorted_versions_array = array ();
+
+    // Constructor
```

Comment:
Remove all un-needed remarks.
It is clear that this is constructor and that are members ,,fucntions ..etc.